cmd/internal/switcher/template.go

```diff
@@ -3,18 +3,20 @@ package switcher
 import (
 	"embed"
 	"path"
+	"text/template"
 )
 
 //go:embed tpl
 var templates embed.FS
 
-var interfacesTPL = mustReadTpl("interfaces.tpl")
-var frrTPL = mustReadTpl("frr.tpl")
-
-func mustReadTpl(tplName string) string {
-	contents, err := templates.ReadFile(path.Join("tpl", tplName))
+func parseFileOrFallback(path string, fallbackFS string) *template.Template {
+	tpl, err := template.ParseFiles(path)
 	if err != nil {
-		panic(err)
+		return mustParseFS(fallbackFS)
```

Comment:
For the user who specifies a template explicitly this behavior would be pretty unexpected. I'd vote for letting the app die.